packages/desktop-client/src/components/mobile/accounts/AccountTransactions.tsx

```diff
@@ -277,10 +276,6 @@ function TransactionListWithPreviews({
         ) {
           reloadTransactions();
         }
-
-        if (tables.includes('payees') || tables.includes('payee_mapping')) {
```

Comment:
No need for this since this is already handles by the global sync-event listener